translations/japanese/core-commands.ts

```diff
@@ -0,0 +1,132 @@
+		"You already extracted the battle input log.": "あなたはすでにログを出力しています。",
+		"Battle input log re-requested.": "ログの出力が再要求されました。",
+		"Invalid input log.": "不正なログです。",
+		"Your input log contains untrusted code - you must have console access to use it.": "入力ログには信頼されていないコードが含まれています。これを使用するにはコンソールへのアクセス権が必要です。",
+		"This command can only be used in a battle.": "このコマンドはバトル内でのみ使用できます。",
+		"Only players can extract their team.": "プレイヤーのみがチームの詳細を閲覧できます。",
+		"Use a number between 1-6 to view a specific set.": "セットを出力するには1から6の間の数字を指定してください。",
+		"The Pokemon \"${target}\" is not in your team.": "\"${target}\"はあなたのチームの中にはいません。",
+		"That Pokemon is not in your team.": "そのポケモンはあなたのチームの中にはいません。",
+		"View team": "チームを見る",
+		"Must be in a battle room.": "バトルの中でなければいけません。",
+		"This server does not allow offering ties.": "このサーバーでは引き分けの提案が許可されていません。",
+		"You can't offer ties in tournaments.": "トーナメントでは引き分けを提案できません。",
+		"It's too early to tie, please play until turn 100.": "引き分けの提案が早すぎます。100ターンまで待ってください。",
+		"No other player is requesting a tie right now. It was probably canceled.": "他のプレイヤーは引き分けを提案していません。おそらくキャンセルされました。",
+		"${user.name} is offering a tie.": "${user.name}が引き分けを提案しています。",
+		"Accept tie": "引き分けの提案を受け入れる",
+		"Reject": "引き分けの提案を拒否する",
+		"Must be a player to accept ties.": "引き分けを受け入れるにはプレイヤーである必要があります。",
+		"You have already agreed to a tie.": "すでに引き分け同意しています。",
+		"${user.name} accepted the tie.": "${user.name}が引き分けに同意しました。",
+		"All players have accepted the tie.": "全てのプレイヤーが引き分けに同意しました。",
+		"Must be a player to reject ties.": "引き分けを拒否するにはプレイヤーである必要があります。",
+		"${user.name} rejected the tie.": "${user.name}が引き分けを拒否しました。",
+		"This room doesn't have an active game.": "この部屋には、進行中のゲームがありません。",
+		"This kind of game can't be forfeited.": "この種類のゲームは降参できません。",
+		"This game doesn't support /choose": "このゲームでは\"/choose\"が使えません。",
+		"This game doesn't support /undo": "このゲームでは\"/undo\"が使えません。",
+		"You can only save replays for battles.": "リプレイが保存できるのはバトルだけです。",
+		"This battle can't have hidden replays, because the tournament is set to be forced public.": "トーナメントの設定でこのバトルを隠し部屋にすることが許されていません。",
+		"The replay for this battle is already set to hidden.": "このバトルのリプレイはすでに隠されています。",
+		"${user.name} hid the replay of this battle.": "${user.name}がこのバトルのリプレイを非表示にしました。",
+		"You can only do this in battle rooms.": "これはバトル部屋のみで使えます。",
+		"You can only add a Player to unrated battles.": "プレイヤーを追加できるのはレートが有効でないバトルのみです。",
+		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "プレイヤーは\"p1\"か\"p2\"に設定しなければなりません。\"${target}\"ではありません。",
+		"User ${name} not found.": "ユーザー ${name} は見つかりませんでした。",
+		"User ${name} must be in the battle room already.": "ユーザー\"${name}\"はすでにバトル部屋に入室しています。",
+		"This room already has a player in slot ${target}.": "スロット${name}にはすでにユーザーがいます。",
+		"${targetUser.name} is already a player in this battle.": "\"${targetUser.name}\"はすでにプレイヤーになっています。",
+		"${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",
```

Comment:
If it is true, I'll delete this line.
Is it true?
<img src="https://user-images.githubusercontent.com/86363603/148243406-f24bd531-5663-4380-b006-0a02781f7d68.jpeg" width="300px">